src/pages/workspace/content/submitted/index.tsx

```diff
@@ -87,15 +100,28 @@ const SubmittedForReviewPage = () => {
     setSortBy(sortBy);
   };
 
- 
-
   useEffect(() => {
     const getReviewContentList = async () => {
       try {
         setLoading(true);
         const query = debouncedSearchTerm || "";
-        const offset =debouncedSearchTerm!==""? 0 : page * LIMIT;
-        const primaryCategory = filter.length ? filter : [];
+        let offset = debouncedSearchTerm !== "" ? 0 : page * LIMIT;
+        const localSelectedFilters = localStorage.getItem("selectedFilters");
+        const selectedFilters = localSelectedFilters
+          ? JSON.parse(localSelectedFilters)
+          : null;
```

Comment:
@AkshataKatwal16 code is repeated everywhere... try to optimize this

Reply:
set by route query now